lib/widgets/caregiver/child_list_element.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+
+import 'package:fokus/utils/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
+
+import 'package:fokus/widgets/child_wallet.dart';
+
+class ChildListElement extends StatelessWidget {
+	final String name;
+	final int todayPlanCount;
+	final int pointCount;
+
+	ChildListElement({
+		@required this.name,
+		@required this.todayPlanCount,
+		@required this.pointCount
+	});
+
+	Image headerImage() {
+		// TODO Handling the avatars (based on type and avatar parameters), returning sunflower for now
+		return Image.asset('assets/image/sunflower_logo.png', height: 80);
+	}
+
+	@override
+	Widget build(BuildContext context) {
+		return Card(
+			shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(AppBoxProperties.roundedCornersRadius)),
+			margin: EdgeInsets.symmetric(
+				vertical: AppBoxProperties.cardListPadding,
+				horizontal: AppBoxProperties.screenEdgePadding
+			),
+			child: IntrinsicHeight(
+				child: Row(
+					mainAxisAlignment: MainAxisAlignment.spaceBetween,
+					crossAxisAlignment: CrossAxisAlignment.start,
+					children: <Widget>[
+						Row(
+							children: <Widget>[
+								Padding(
+									padding: EdgeInsets.symmetric(horizontal: 6.0, vertical: 4.0),
+									child: headerImage()
+								),
+								Column(
+									crossAxisAlignment: CrossAxisAlignment.start,
+									children: <Widget>[
+										Padding(
+											padding: EdgeInsets.only(top: 6.0),
+											child: Text(
+												name,
+												style: Theme.of(context).textTheme.headline3
+											)
+										),
+										Text(
+											AppLocales.of(context).translate('page.caregiverSection.panel.content.plansTodayCount', {'NUM_PLANS': todayPlanCount.toString()}) + ' ' +
+											AppLocales.of(context).translate('page.caregiverSection.panel.content.plansToday'),
+											style: Theme.of(context).textTheme.subtitle2
+										),
+										Padding(
+											padding: EdgeInsets.only(bottom: 4.0),
+											child: Row(
+												children: <Widget>[
+													ChildPointsChip(quantity: pointCount)
+												],
+											)
+										),
+									],
+								)
```

Comment:
I may be going to far with pattern matching :smile: but that column arrangement of big title, small description and some item also repeats a couple of times (like in Child plans)

Reply:
Advanced custom widget ItemCard for 90% of the prototyped lists.
For use in universal Segment section.

Reply:
https://www.youtube.com/watch?v=RQdxHi4_Pvc

Reply:
```
   ^
  / \
 /   \
/     \
|  |  |
|  |  |
\  \  /
 \  \/
 /\  \
/  \  \
|  |  |
|  |  |
\     /
 \   /
  \ /
   v   egment
```